### ISSUE TITLE
Resuming a multi order by query from a continuation token

### ISSUE THREAD

**Issue by apescione** [Azure/azure-sdk-for-java#10010]
Pagination of a query with a multi order from a continuation token

In code is written that an implementation in .NET Sdk could work fine if ported into JAVA SDK.

How can I do my query/quiries to have the same result before having the new Feature?

Unfortunally, the old client 3.1.0 didn't rise same error, so we used that approach (Probabily with wrong results)

com.azure.data.cosmos.internal.query.OrderByDocumentQueryExecutionContext<T>
Code that rises the exception:

```
      // This code path needs to be implemented, but it's error prone and needs
            // testing.
            // You can port the implementation from the .net SDK and it should work if
            // ported right.
            throw new NotImplementedException(
                    "Resuming a multi order by query from a continuation token is not supported yet.");
```

**Comment by baldassarreC** [comment]
I catch a "NotImplementedException" with azure-cosmos v3.7.0. Maybe it's the same issue.
Is there a new library version to solve?

**Comment by joshfree** [comment]
@kushagraThapar could you please follow up on this customer-reported cosmos sdk issue?

**Comment by kushagraThapar** [comment]
@apescione @baldassarreC - Thanks for reporting this issue. @mbhaskar can you please help us figure out if this feature is still in our backlog ?

**Comment by kushagraThapar** [comment]
@apescione @baldassarreC sorry for the delay on this issue, we are investigating it now. 
@xinlian12 - I am not sure if we even need to throw this exception anymore based on the recent query fixes that you have done. 
Let's discuss this issue.

**Comment by kushagraThapar** [comment]
@apescione @baldassarreC - We investigated this issue and this is in our backlog for this semester. We will plan it soon and will update this issue on the timeline.

**Comment by kushagraThapar** [comment]
This is a feature request that is currently in plan for this semester. 
@mbhaskar  - can you please share some insights on this issue ?

**Comment by kushagraThapar** [comment]
@mbhaskar - can you please provide an update on this issue ?

**Comment by chandan-udaan** [comment]
We're also facing the same issue and have similar use-case to show documents to client on a multi-order using continuation token
`! Causing: org.apache.commons.lang3.NotImplementedException: Resuming an order by query from a continuation token is not supported yet.
! at com.microsoft.azure.cosmosdb.rx.internal.query.OrderByDocumentQueryExecutionContext.initialize(OrderByDocumentQueryExecutionContext.java:167)`

@kushagraThapar @mbhaskar 
Do you have an idea by when this feature is planned to be rolled out?

**Comment by mbhaskar** [comment]
This feature is in the backlog. Current tentative plan is to have this supported by mid of this year

**Comment by kushagraThapar** [comment]
@mbhaskar  - I believe this has already been addressed. If so, please close the issue, thanks.

**Comment by xinlian12** [comment]
Has been released >= 4.26.0
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos/CHANGELOG.md#4260-2022-02-11
